modules/configuring-default-seccomp-profile.adoc

```diff
@@ -1,15 +1,99 @@
 [id="configuring-default-seccomp-profile_{context}"]
-= Configuring the default seccomp profile
-OpenShift ships with a default seccomp profile that is referenced as `runtime/default`. You can enable the default seccomp profile for a pod or container workload by setting `RuntimeDefault` as following:
+= Enabling the default seccomp profile for all {product-title} pods
+{product-title} ships with a default seccomp profile that is referenced as `runtime/default`. In {product-version} newly created pods have the Security Context Constraint (SCC) set to `restricted-v2` and the default seccomp profile applies to the pod. 
 
-.Example
 
-[source, yaml]
+. You can verify Security Context Constraint (SCC) set on a pod by running the following commands: 
+
+.. Verify what pods are running in the namespace: 
++
+[source, terminal]
+----
+$ oc get pods -n <namespace> 
+---- 
++
+For example to verify what pods are running in the `workshop` namespace run the following: 
++
+[source, terminal]
+----
+$ oc get pods -n workshop 
+---- 
++
+.Example output 
++
+[source, terminal]
+----
+NAME                READY   STATUS      RESTARTS   AGE
+parksmap-1-4xkwf    1/1     Running     0          2m17s
+parksmap-1-deploy   0/1     Completed   0          2m22s
+---- 
++
+.. Inspect the pods: 
++
+[source, terminal]
+----
+$ oc get pod parksmap-1-4xkwf -n workshop -o yaml
+----
++
+.Example output 
++
+[source, terminal]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/network-status: |-
+      [{
+          "name": "openshift-sdn",
+          "interface": "eth0",
+          "ips": [
+              "10.131.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      }]
+    k8s.v1.cni.cncf.io/networks-status: |-
+      [{
+          "name": "openshift-sdn",
+          "interface": "eth0",
+          "ips": [
+              "10.131.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      }]
+    openshift.io/deployment-config.latest-version: "1"
+    openshift.io/deployment-config.name: parksmap
+    openshift.io/deployment.name: parksmap-1
+    openshift.io/generated-by: OpenShiftWebConsole
+    openshift.io/scc: restricted-v2 <1>
+    seccomp.security.alpha.kubernetes.io/pod: runtime/default <2>
+---- 
+<1> The `restricted-v2` is set by default on newly created pods in {product-version}. 
+<2> The `restricted-v2` sets the seccompProfile to `runtime/default` by default. 
+
+You can enable the default seccomp profile for a pod or container workload that does not have the seccompProfile set by adding `restricted-v2` to the default service account. 
+
+Follow these steps:
+
+. Add the `restricted-v2` to the ServiceAccount: 
```

Comment:
As mentioned above, this is not required, all authenticated users have access to the SCC.

Reply:
Hi @mvazquezc I guess I need some clarification on "all authenticated users have access to the SCC" what does that mean exactly. All pods have `restricted-v2` set by default

Reply:
There is a group in K8s named `system:authenticated`. In OCP 4.10 and below the `restricted` SCC was available for this group. In a fresh install of OCP 4.11, this group will have access to the `restricted-v2` SCC instead.

And what SCC a pod gets assigned depends on the SCCs that the user creating that pod has access to, you can read more about that [here](https://docs.google.com/presentation/d/1mwqYQP-HRaCgk317F5IIlvhRYfVBC5lKbFSu5KATXmM/edit#slide=id.gbca891d708_0_11).